tests/virtualization/universal/hotplugging_HDD.pm

```diff
@@ -28,15 +28,16 @@ sub test_add_virtual_disk {
     my $disk_image = get_disk_image_name($guest, $disk_format);
 
     assert_script_run("rm -f $disk_image");
-    assert_script_run "qemu-img create -f $disk_format $disk_image 10G";
+    # Set disk size=9.5G to make it to be found more easily within the guest
+    assert_script_run "qemu-img create -f $disk_format $disk_image 9.5G";
     my $domblk_target = 'vdz';
     $domblk_target = 'xvdz' if (is_xen_host);
     script_run("virsh detach-disk $guest ${domblk_target}", 240);
     if (try_attach("virsh attach-disk --domain $_ --source $disk_image --target ${domblk_target}")) {
         assert_script_run "virsh domblklist $guest | grep ${domblk_target}";
         assert_script_run("ssh root\@$guest lsblk");
         # Attach disk check
-        assert_script_run("ssh root\@$guest lsblk | grep -iE '[x]?vd[b-z]'", timeout => 60, fail_message => "Failed to attach disk for guest $guest");
+        assert_script_run("ssh root\@$guest lsblk | grep -iE '[x]?vd[a-z] +.*9.5G'", timeout => 60, fail_message => "Failed to attach disk for guest $guest");
```

Comment:
For earlier release, the default disk is vda, right? Then for released products, in MU test, this assert will be problematic if vda is found only? @RoyCai7  Please be aware.

Reply:
I think is ok

Reply:
Get the change info.

Reply:
> In the failed tests the size of vda is 10G, in the VR is 9.5G. I think the expression should be searching disks that are 10G or less?

The attached disk is change to 9.5G too:
`assert_script_run "qemu-img create -f $disk_format $disk_image 9.5G";`

The figure "9.5G" is to flag that this is the exact attached disk within the guest vm, if we could remember to not install a guest with 9.5G size of storage initially in our tests.

Reply:
I haven't notice that, sorry. The change looks good to me :)